lms/templates/help_modal.html

```diff
@@ -208,7 +202,7 @@
     cycle_modal_tab("#feedback_submit", "#feedback_form_wrapper .close-modal");
     cycle_modal_tab("#feedback-faq-link", "#feedback_success_wrapper .close-modal");
 % else:
-    cycle_modal_tab("#howto-enroll-link", "#help_wrapper .close-modal");
+    cycle_modal_tab("#last-link", "#help_wrapper .close-modal");
```

Comment:
Yeah, we should try to make this a class (and do a `$.find` within the modal) because I can imagine a world where:
- another element is added to the page with a duplicate ID
- we want multiple modals to exist in the DOM concurrently